stories/ui-search/search.stories.ts

```diff
@@ -78,12 +78,10 @@ export const autoSubmit = () => ({
   component: TsSearchComponent,
   props: {
     autoSubmit: true,
+    debounceTime: number('Debounce Time', 200),
     submitted: action('Submitted'),
   },
 });
-autoSubmit.parameters = {
-  knobs: { disabled: true },
-};
```

Comment:
@doctorssm what's the reason that this block is removed?

Reply:
I have added deboune time knob. So by deleting code above I activate this knob in story book
![M4FcjIHshN](https://user-images.githubusercontent.com/10260908/108745181-75876c00-7543-11eb-866d-e4e60e426428.png)